code/modules/reagents/reagents/medicine.dm

```diff
@@ -609,7 +609,7 @@
 	..()
 	if(prob(5))
 		M.emote(pick("twitch", "blink_r", "shiver"))
-	M.add_chemical_effect(CE_SPEEDBOOST, 1)
+	M.add_chemical_effect(CE_SPEEDBOOST, 1.2)
```

Comment:
I personally feel like this is fine but others might disagree. Also hyperzine only really works in increments of 0.5 from what I've been told. I'd say to just bump it back down to only 1.0